cmake/therock_subproject.cmake

```diff
@@ -971,19 +999,19 @@ function(_therock_cmake_subproject_setup_toolchain
     get_target_property(_amd_llvm_dist_dir "${_toolchain_subproject}" THEROCK_DIST_DIR)
     get_target_property(_amd_llvm_stamp_dir "${_toolchain_subproject}" THEROCK_STAMP_DIR)
     # Add a dependency on the toolchain's dist
-    set(AMD_LLVM_C_COMPILER "${_amd_llvm_dist_dir}/lib/llvm/bin/clang")
-    set(AMD_LLVM_CXX_COMPILER "${_amd_llvm_dist_dir}/lib/llvm/bin/clang++")
-    set(AMD_LLVM_LINKER "${_amd_llvm_dist_dir}/lib/llvm/bin/lld")
+    set(AMD_LLVM_C_COMPILER "${_amd_llvm_dist_dir}/lib/llvm/bin/clang${EXE_SUFFIX}")
```

Comment:
Is there a reason to not use CMAKE_EXECUTABLE_SUFFIX ?

Reply:
Ah, right. That does exist in CMake. I'll use it. I think _Python_ doesn't have such operating system prefix/suffix utilities?

Reply:
🤦 maybe I did a search in subfolders and found https://github.com/ROCm/llvm-project/blob/202d9a7b51f3669149ebc442f20185883fd0ba82/amd/device-libs/cmake/OCL.cmake#L22-L26

Should have known better :P